Resources/Textures/_Harmony/Objects/Specific/Hydroponics/Firelemon.rsi/meta.json

```
@@ -0,0 +1,50 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "lemon sprites made from https://github.com/Baystation12/Baystation12/commit/7edcb92d087d0cb0ed17174aea148840be675dc5, drink sprites made by Ermucat",
```

As always, I'm going to nitpick attributions.
```suggestion
  "copyright": "Lemon sprites taken from baystation at commit https://github.com/Baystation12/Baystation12/commit/7edcb92d087d0cb0ed17174aea148840be675dc5. Drink sprites made by Ermucat (GitHub)",
```